CONTRIBUTING.md

```diff
@@ -30,7 +30,7 @@ I do not use Eclipse. If you have instructions to compile with Eclipse, please e
 
   * We use tab and not space. Be sure to disable "expand tabs to space" in your IDE.
   * Swing JComponent variable names should have leading "j". Ex.: jMyTable
-  * No other variable names should use Systems Hungarian
+  * No other variable names should use [Systems Hungarian](https://en.wikipedia.org/wiki/Hungarian_notation)
```

Comment:
@GoldenGnu are you actually using this?

Reply:
Oh, "no other", misread that :+1:

Reply:
It's fine, I was just confused. I missed the "no other" part and thought this is how all variables should be, but couldn't actually find any names with this pattern :slightly_smiling_face: